index.html

```diff
@@ -41,24 +45,36 @@
 	//zoomToPolygon() zooms the map to the district extent
 
 			function runWhenLoadComplete() {
-				if (!map.loaded()) {
+				if (!map.loaded() || !map.getLayer('texas-school-districts-poly')) {
 					setTimeout(runWhenLoadComplete, 100);
 				}
 				else {
 					populateZoomControl("school-districts-control", "texas-school-districts", "NAME", "Texas School Districts");
+					map.moveLayer('texas-school-districts-lines', 'country-label-sm');
+					map.moveLayer('texas-school-districts-poly', 'texas-school-districts-lines');
+					for (i=0; i < loadedLineLayers.length; i++) {
+						if (loadedLineLayers[i][1] !== "texas_school_districts") {
+							map.moveLayer(loadedLineLayers[i][0], 'texas-school-districts-poly');
+							map.moveLayer(loadedPolygonLayers[i][0], loadedLineLayers[i][0]);
+						}
+					}
 				}
 			}
 
 			function populateZoomControl(selectID, sourceID, fieldName, layerName) {
 				polygons = getPolygons(sourceID, fieldName);
 				var select = document.getElementById(selectID);
-				select.options[0] = new Option(layerName, "-108,25,-88,37");
+				select.options[0] = new Option(layerName, "-108,25,-88,37,whole state of Texas");
```

Comment:
The corny name here is because I needed something I could be sure that no school district would be called, so that the `zoomToPolygon()` function will later be able to tell when we're zooming out instead of zooming to a specific district.

Reply:
Should have gone for Deep in the Heart of Texas ;)

Reply:
There's nothing stopping us....